plugins/inventory/libvirt.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+name: libvirt
+plugin_type: inventory
+extends_documentation_fragment:
+    - constructed
+short_description: Libvirt inventory source
+description:
+    - Get libvirt guests in an inventory source
+author:
+    - Dave Olsthoorn <dave@bewaar.me>
+version_added: "2.10"
+options:
+    plugin:
+        description: token that ensures this is a source file for the 'libvirt' plugin.
+        required: True
+        choices: ['libvirt']
+    connection:
+        description: Libvirt Connection URI
+        required: True
+        type: str
```

Comment:
I would like this if it could be a list, so that it could gather from multiple hosts instead of just one. That could be a follow on bit of work though.

Reply:
that should already be possible with [multiple inventories](https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html#using-multiple-inventory-sources) tough... so should there be explicit support for this?

Reply:
Oh! good point! That totally makes sense. Never mind then. :)